src/app/(client)/meal/post/edit/components/edit-result-section.tsx

```
@@ -90,76 +98,36 @@ const EditResultSection = ({ imageList, initialMealList }: EditResultSectionProp
-      } catch (error) {
-        console.error('식사 정보 생성 중 오류 발생:', error);
-      }
+      const newMeals = { foodImages: storedImageUrls, ateAt, mealCategory, memo };
+      await Promise.all([createMeal(newMeals, mealList), deleteAnalysis()]);
+      router.push(SITE_MAP.HOME);
     } catch (error) {
-      alert('식사 정보 등록에 실패했습니다. 잠시 후 다시 시도해주세요.');
-      console.error('식사 정보 등록 실패:', error);
-      router.refresh();
+      const isKnownError = typeof error === 'object' && error !== null && 'title' in error && 'description' in error;
+      error = isKnownError ? error : ERROR_MESSAGES.AI_ANALYSIS_FAILED_DEFAULT;
```

Biome (1.9.4)

[error] Reassigning a catch parameter is confusing.

The catch parameter is declared here:

Use a local variable instead.

(lint/suspicious/noCatchAssign)